### ISSUE TITLE
Blackfathom Deeps - Altar of the Deeps

### ISSUE THREAD

**Issue by adellaci** [unified-db/Database#164]
In Blackfathom Deeps, in Aku'mai's Lair (x=-839.290283,y=-478.257751,z=-30.712105), if you click on the Altar of the Deeps it should teleport Alliance players to Darnassus(x=10073.755859,y=2356.2573.24,z=1331.892822,o=4.0)

**Comment by Grz3s** [comment]
hi ..thanks for report
i have found some good info for this:
```quote
By mjolnir82991 (867 – 7)  on 2007/08/16 (Patch 2.1.3)   
The Altar's changed location warps you to Darnassus. As well as giving you the Blessing of Blackfathom for a hour.
```
and this one
```quote
If you're Horde, it just gives you the debuff but doesn't teleport you away.
```
also I'm sure that buff  = ID - 8733 Blessing of Blackfathom
```code
Effect 0: Id 6 (SPELL_EFFECT_APPLY_AURA)
BasePoints = 5
Targets (25, 0) (TARGET_DUELVSPLAYER, NO_TARGET)
Aura Id 29 (SPELL_AURA_MOD_STAT), value = 5, misc = 3 (UNIT_MOD_STAT_INTELLECT), miscB = 0, periodic = 0

Effect 1: Id 6 (SPELL_EFFECT_APPLY_AURA)
BasePoints = 5
Targets (25, 0) (TARGET_DUELVSPLAYER, NO_TARGET)
Aura Id 29 (SPELL_AURA_MOD_STAT), value = 5, misc = 4 (UNIT_MOD_STAT_SPIRIT), miscB = 0, periodic = 0

Effect 2: Id 6 (SPELL_EFFECT_APPLY_AURA)
BasePoints = 15
Targets (25, 0) (TARGET_DUELVSPLAYER, NO_TARGET)
Aura Id 13 (SPELL_AURA_MOD_DAMAGE_DONE), value = 15, misc = 16 (MASK_FROST), miscB = 0, periodic = 0
```
and question:
-  how this altar worked? all the time or only after boss was killed?
-  Darnasus xyz - where this info comming from?

**Comment by adellaci** [comment]
If I remember correctly it worked all the time.
My xyz may not be exact, but I am sure that it teleported you to the NPC that gives one of the quest in Blackfathom Deeps.


Sent from my Verizon Wireless 4G LTE smartphone

-------- Original message --------

**Comment by adellaci** [comment]
Hi Grz3s
I did more research. If you have the quest 'Blackfathom Villainy', you kill 'Aku'mai', you are of the Alliance faction, and you click on the altar, you receive 'Blessing of Blackfathom' for a hour, and  'Morridune' appears from the far right of the altar, walking towards the altar and yells;
```
 Aku'mai is dead! At last, I can leave this wretched place.
```
When he arrives at the altar he says;
```
Speak with me to hear my tale.
```
If you click upon him, he tells his story;
```
"I once followed the ways of the Twilight's Hammer. Obsessed with reviving the Old Gods, I journeyed deep into this lair to try to draw power from the cursed beast, Aku'mai!

After witnessing the sacrifice of countless innocents, I came to my senses.

I was trapped but now I am free! But to far from my homeland of Khaz Modan!

Brave adventurer, I offer you passage to a very special land in return for slaying Aku'mai. If you wish, I can port you to the city of Darnassus."
```
At the end of the text there is a place to click /w a speak bubble to the left of it;
```
Please port me to Darnassus.
```
I am pretty sure that if one person in the party has the quest, all party members can use the port.
Also, if no one in the group has the quest you get the buff but 'Morridune' will not appear.

Sources:
http://wowwiki.wikia.com/wiki/Morridune
http://www.wowhead.com/npc=6729/morridune#comments
http://www.wowhead.com/npc=6729/morridune#screenshots:id=156418

More information:
Patch 4.0.3a (2010-11-23): Removed.
Doesn't work for the Horde faction, and they received a debuff as shows here;
https://www.youtube.com/watch?v=UyyUM1rZras&spfreload=10
